service/controller/v3/key/key.go

```diff
@@ -200,6 +202,72 @@ func ToCustomObject(v interface{}) (providerv1alpha1.AzureConfig, error) {
 	return customObject, nil
 }
 
+func ToKeyValue(m map[string]interface{}) (interface{}, error) {
```

Comment:
We need a lot of helper methods in order to work against the deployment's version blobs.